docker/fuzzbench/Dockerfile

```diff
@@ -22,9 +22,16 @@ RUN wget https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKER_
 
 WORKDIR /fuzzbench
 
-ADD requirements.txt /fuzzbench/requirements.txt
+COPY requirements.txt requirements.txt
 RUN pip3 install -r requirements.txt
 
-ADD fuzzbench/*.py /fuzzbench/
+COPY alembic.ini alembic.ini
```

Comment:
I think alembic.ini is probably not needed as it is used only in development (it is used for schema migrations in SQL).

Reply:
Right, we'd need it to initialize the db schema for new db instances, e.g., at the beginning of the e2e test we'd use an ephemeral postgres container, or when setting up a postgres server for local experiments.

Reply:
Are we sure about the db being ephemeral? I thought we seed this is TBD (my personal thought is that this isn't a good idea (unlike redis which can be ephemeral)).

Also, I don't think you need it for initializing a new db. Just for migrating between schemas. This is how local experiments work. In dispatcher.py there is code to initialize the db that can be done by sqlalchemy and without alembic.

(I'm not horribly concerned about this file polluting the image btw, just trying to clarify things :-)

Reply:
> Are we sure about the db being ephemeral? I thought we said this is TBD (my personal thought is that this isn't a good idea (unlike redis which can be ephemeral) I don't see how it would work with the fuzzbench service also).

The service of course would not use an ephemeral db, but eg. the e2e test running in ci (ie. it'd create a fresh postgres docker container with e2e-test docker compose config).

> Also, I don't think you need it for initializing a new db. Just for migrating between schemas. This is how local experiments work. In dispatcher.py there is code to initialize the db that can be done by sqlalchemy and without alembic.

Ah cool! So that'd create the tables too in a fresh postgres instance (eg used during the e2e test), before inserting rows?